bundle-generator/deployment/src/test/java/io/quarkiverse/operatorsdk/bundle/sources/FirstReconciler.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkiverse.operatorsdk.bundle.sources;
+
+import io.javaoperatorsdk.operator.api.reconciler.Context;
+import io.javaoperatorsdk.operator.api.reconciler.Reconciler;
+import io.javaoperatorsdk.operator.api.reconciler.UpdateControl;
+import io.quarkiverse.operatorsdk.bundle.runtime.CSVMetadata;
+
+@CSVMetadata(name = "first-operator")
```

Comment:
I think the default test should actually be to check that we generate a CSV per reconciler in the absence of `CSVMetadata` annotations.

Reply:
added